index.html

```diff
@@ -292,7 +306,17 @@ <h3>
 
             </ul>
         </div>
+    </div>>
     </section>
+    <script src="menu.js">
+        var menu = document.querySelector("navbar_menu");
+        function openmenu() {
+            document.querySelector('body').classList.add('no-scroll')
+        }
+        function closemenu() {
+            document.querySelector('body').classList.remove('no-scroll')
+        }
+        
+    </script>
```

Comment:
- [ ] You have done great so far, your mobile menu works great :+1: :100: 
- - But following the best practices, you are not meant to have javascript code inside your HTML file. You already have a `menu.js` file. I'll suggest you move the javascript code there. :smile: 
- -  Also currently, on the desktop menu, clicking on the menu item doesn't take you to that section of the page, I'll suggest you update that too. :smile: :small_airplane:

Reply:
Hey, I have implemented requirement changes in menu.js . Kindly I don't think the desktop menu is part of the project requirement but I will implement it.
Thank you.